src/discord.ts

```diff
@@ -1,10 +1,15 @@
 import {MessageEmbed, WebhookClient} from 'discord.js';
 
 import {environment} from './environment/environment';
-import {ExaminedRun, Run} from './srcom';
+import {Flag} from './flags';
+import {ExaminedRun, Run, getPlayerFromRun} from './srcom';
 import {runToString} from './util';
 
 const webhookClient = new WebhookClient(environment.webhookChannelId, environment.webhookSecret);
+const rejectedRunsWebhookClient = new WebhookClient(
+  environment.rejectedRunsWebhookChannelId,
```

Comment:
since we log to two channels now, can you rename `webhookChannelId` and `webhookSecret` to `dailyStatsWebhookChannelId` and `dailyStatsWebhookSecret`?

Reply:
will also need to update `build/create-environment.ts` to generate an environment file with all of these fields. you can update the environment variable names to match the new variable names in the TS code; I'll update the CI config to match